### PR TITLE
Change title on README

### PR DESCRIPTION
**Pull request by ahastudio**
Change title text: `git-sample` -> `Git Sample`.

Write body text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# git-sample
```
Comment (angelxtry) [inline README.md:1]: test

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+# Git Sample Project
```
Comment (angelxtry) [inline README.md]: new test

Comment (ahastudio) [inline README.md]: test!

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-# git-sample
\ No newline at end of file
+# Git Sample
+
+Git Sample Project.
```
Comment (angelxtry) [inline README.md]: 문장으로 쓰면 좋겠어요.

**Review by angelxtry — CHANGES_REQUESTED** [review]
test

**Review by angelxtry — APPROVED** [review]
:+1:

**Review by angelxtry — APPROVED** [review]
:+1:

**Pull request merged**